slime-framework/util/util.go

```diff
@@ -282,3 +308,25 @@ func SetLevel(lvl string) error {
 func SetReportCaller(support bool) {
 	log.SetReportCaller(support)
 }
+
+
+func GetLevel() string {
+	level := log.GetLevel()
+	return level.String()
+}
+
+// SetKlog while x<= KlogLevel in the klog.V("x").info("hello"), log will be record
+func SetKlog(KlogLevel int32) {
+	fs = flag.NewFlagSet("klog",flag.ExitOnError)
+	klog.InitFlags(fs)
```

Comment:
new fs when init log and store it and use the exact one for later setting action